tools/switch.sh

```diff
@@ -260,6 +262,26 @@ if [ $# -lt 1 ]; then
     exit 1
 fi
 
+function ep-tunnel-command() {
+    local SWITCH=$1 CMD=$2
+    echo "Execute switch ep-tunnel $CMD on $SWITCH"
+
+    DRIVES=$(switchtec fabric gfms-dump "$SWITCH" | grep "SRIOV-PF" -A2 | grep PDFID | awk '{print $2}')
```

Comment:
Since this is common to nvme.sh, you may want to make it a function in that file and use it here